packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderMenuIcon.tsx

```diff
@@ -2,20 +2,18 @@ import * as React from 'react';
 // @ts-expect-error fixed in Material-UI v5, types definitions were added.
 import { unstable_useId as useId } from '@material-ui/core/utils';
 import IconButton from '@material-ui/core/IconButton';
-import { gridColumnMenuStateSelector } from '../../hooks/features/columnMenu/columnMenuSelector';
-import { useGridSelector } from '../../hooks/features/core/useGridSelector';
 import { classnames } from '../../utils/classnames';
 import { GridApiContext } from '../GridApiContext';
 import { GridColDef } from '../../models/colDef/gridColDef';
 
 export interface ColumnHeaderFilterIconProps {
   column: GridColDef;
+  open: boolean;
 }
 
 export function ColumnHeaderMenuIcon(props: ColumnHeaderFilterIconProps) {
-  const { column } = props;
+  const { column, open } = props;
```

Comment:
wrong name here `ColumnHeaderFilterIconProps`

Reply:
I'll open another PR to rename it.

Reply:
Not sure it's worth it. Simple fix. Just making the code cleaner while you're here.

Reply:
I renamed it. I was thinking about not changing things out of scope of this PR but this one was easy and without impacts. 👍